Description: Remove upstream's `-O3` build optimization
 This permits to build with Debian's default optimization which are set through
 buildflags.mk
 .
 With this, usually it will build with -O2.
 .
 Using this mechanism also permits to honor the DEB_BUILD_OPTIONS=noopt in
 which case, optimization is set to -O0.
 .
 References:
  - https://wiki.debian.org/HardeningWalkthrough
  - https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html
 .
Author: Fab Stz <fabstz-it@yahoo.fr>
Forwarded: not-needed
Last-Update: 2024-02-08

--- a/FreeFileSync/Source/Makefile
+++ b/FreeFileSync/Source/Makefile
@@ -6,7 +6,7 @@
 
 CXXFLAGS += -std=c++23 -pipe -DWXINTL_NO_GETTEXT_MACRO -I../.. -I../../zenXml -include "zen/i18n.h" -include "zen/warn_static.h" \
            -Wall -Wfatal-errors -Wmissing-include-dirs -Wswitch-enum -Wcast-align -Wnon-virtual-dtor -Wno-unused-function -Wshadow -Wno-maybe-uninitialized \
-           -O3 -DNDEBUG `$(WX_CONFIG) --cxxflags --debug=no` -pthread
+           -DNDEBUG `$(WX_CONFIG) --cxxflags --debug=no` -pthread
 
 LDFLAGS += `$(WX_CONFIG) --libs std, aui, richtext --debug=no` -pthread
 
--- a/FreeFileSync/Source/RealTimeSync/Makefile
+++ b/FreeFileSync/Source/RealTimeSync/Makefile
@@ -6,7 +6,7 @@
 
 CXXFLAGS += -std=c++23 -pipe -DWXINTL_NO_GETTEXT_MACRO -I../../.. -I../../../zenXml -include "zen/i18n.h" -include "zen/warn_static.h" \
            -Wall -Wfatal-errors -Wmissing-include-dirs -Wswitch-enum -Wcast-align -Wnon-virtual-dtor -Wno-unused-function -Wshadow -Wno-maybe-uninitialized \
-           -O3 -DNDEBUG `$(WX_CONFIG) --cxxflags --debug=no` -pthread
+           -DNDEBUG `$(WX_CONFIG) --cxxflags --debug=no` -pthread
 
 LDFLAGS += `$(WX_CONFIG) --libs std, aui, richtext --debug=no` -pthread
 
